### DISCUSSION TITLE
Translating src/index.md

### DISCUSSION THREAD

**Discussion by anaclumos** [facebook/docusaurus#10365] [General] [answered]
How can I translate `src/index.md`?

All the docs mention how to translate docs and blog, but not the markdown files in the `src` folder.

How can I do it?

**Comment by Josh-Cena** [comment, marked as answer]
Hi! `src/index.md` is not a content location by default. Do you mean `src/pages/index.md`? If so please see https://docusaurus.io/docs/i18n/tutorial#translate-the-pages
